taichi/python/export_ggui.cpp

```diff
@@ -218,11 +223,13 @@ struct PyScene {
                      py::tuple color,
                      bool two_sided,
                      FieldInfo transforms,
+                     float draw_instance_count,
                      float draw_first_instance,
                      float draw_index_count,
```

Comment:
nit: I somehow still prefer these counts to be of int type (esp. in C++) to avoid confusing developers, but we can discuss those separately and address that in a different PR if applicable

Reply:
Ok, maybe we can talk about this deeply later.